drivers/misc/rwbuffer.c

```diff
@@ -68,58 +68,26 @@ static ssize_t rwb_read_(FAR struct rwbuffer_s *rwb, off_t startblock,
  ****************************************************************************/
 
 /****************************************************************************
- * Name: rwb_semtake
+ * Name: rwb_lock
  ****************************************************************************/
 
 #if defined(CONFIG_DRVR_WRITEBUFFER)
-static int rwb_semtake(FAR sem_t *sem)
+static int rwb_lock(FAR mutex_t *lock)
 {
-  return nxsem_wait_uninterruptible(sem);
+  return nxmutex_lock(lock);
```

Comment:
Why not just `#  define rwb_lock(l) nxmutex_lock(l)`? the same as for `rwb_unlock`

Reply:
Done